### PR TITLE
Feat : Homepage

### PR DESCRIPTION
**Pull request by yogesh-**
Completed homepage layout and design.

image

### CODE REVIEW

**Review by NtshVrm — APPROVED** [review]

**Comment by NtshVrm** [comment]
Looks good👏

File: index.html [hunk 1]
```diff
@@ -5,16 +5,197 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>sneakerDON</title>
+    <title>sneakers</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://agniui.netlify.app/styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+
 
 </head>
```
Comment (Mohasin-Haque) [inline index.html:14]: remove spaces

File: index.html [hunk 2]
```diff
@@ -5,16 +5,197 @@
...
 
 <body>
 
-    <nav>
+    <div class="home-page">
+
```
Comment (Mohasin-Haque) [inline index.html:19]: Needs some formatting

File: index.html [hunk 3]
```diff
@@ -5,16 +5,197 @@
...
+                            <div class="number-red">12</div>
+                        </div>
+
+                    </li>
+
+
+
+                    <!-- <li><a href="news.asp">News</a></li>
+                <li><a href="contact.asp">Contact</a></li>
+                <li><a href="about.asp">About</a></li> -->
```
Comment (Mohasin-Haque) [inline index.html:64]: remove commented code

**Review by Mohasin-Haque — COMMENTED** [review]
Your code seems good to me but needs to work on the comments. Please go through it once,

**Pull request merged**